include/cudaq/Target/OpenQASM/OpenQASMEmitter.h

```diff
@@ -14,8 +14,9 @@
 
 namespace cudaq {
 
-/// Translates the given operation to OpenQASM code. The operation or operations
-/// in the region of 'op' need almost all be in QTX dialect.
+/// Translates the given operation to OpenQASM code. The operation, `op`,  or
+/// operations in its region must be in Quake memory reference form. Also,
+/// qubit vectors cannot have unknown size.
```

Comment:
FYI: I've been eliminating the use a "qubit" in the IR documentation and instead using "quantum reference" and/or "wire" as appropriate. "qubit" seemed a bit too slippery in the IR. (Admittedly, the ambiguity might be exactly what we want here?)